test/xebow_web/live/animations_live_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule XebowWeb.AnimationsLiveTest do
+  use XebowWeb.ConnCase
+
+  import Phoenix.LiveViewTest
+
+  test "shows available animations", %{conn: conn} do
+    {:ok, view, _html} = live(conn, Routes.animations_path(conn, :index))
+
+    assert view
+           |> element("#animation_breathing")
+           |> has_element?()
```

Comment:
:warning: (recommended)

This is closely coupled with the list of animations that are hard-coded into the modules. Since we eventually want to change that, we should probably log an issue to improve this test and be more careful with how we mock/control the animations that get loaded into the application during testing.

Reply:
>This is closely coupled with the list of animations that are hard-coded into the modules.

Yeah, I thought about this one for a while before finally going with this. Because it's coupled, I only added one small test around this so it's easy to change in the future. If you can think of a good way to write this that decouples it, I'm open to suggestions.

We could dependency inject the animations, but the overhead that comes with that feels less pragmatic than just changing this single element name if the breathing animation is ever removed. Also, in the next phase of the animations page this test will probably be irrelevant as the page will need to store custom configurations for animations. That could be a good time for more robust testing here, since this page will have complex logic at that point.

>Since we eventually want to change that, we should probably log an issue to improve this test

I think we can leave it up to the assignee of the next feature ticket for this page to do what they want with this test. Right now it's really just a sanity check to make sure a side effect in some other code changing doesn't break the animations page.

Feel free to open a refactor ticket if you feel passionate about this test. Otherwise I think it will naturally be taken care of in the feature work.

Reply:
I kept from suggesting too much because I couldn't think of anything off hand to fix it that didn't seem really involved and possibly over complex. I agree there's not much sense in thorough tests for something that's going to change sooner rather than later. Good enough reasoning to leave it be.